diff --git a/func/lwip_intf/lwip-2.0.2/src/api/sockets.c b/func/lwip_intf/lwip-2.0.2/src/api/sockets.c
index b3696cb57..9457f1c87 100644
--- a/func/lwip_intf/lwip-2.0.2/src/api/sockets.c
+++ b/func/lwip_intf/lwip-2.0.2/src/api/sockets.c
@@ -47,7 +47,7 @@
 #include "lwip/opt.h"
 
 #if LWIP_SOCKET /* don't build if not configured for use in lwipopts.h */
-
+void *os_memset(void *b, int c, UINT32 len);
 #include "lwip/sockets.h"
 #include "lwip/api.h"
 #include "lwip/sys.h"
diff --git a/func/lwip_intf/lwip-2.0.2/src/include/lwip/def.h b/func/lwip_intf/lwip-2.0.2/src/include/lwip/def.h
index 01f462af2..cecb960b8 100644
--- a/func/lwip_intf/lwip-2.0.2/src/include/lwip/def.h
+++ b/func/lwip_intf/lwip-2.0.2/src/include/lwip/def.h
@@ -40,7 +40,7 @@
 /* arch.h might define NULL already */
 #include "lwip/arch.h"
 #include "lwip/opt.h"
-#include "generic.h"
+//#include "generic.h"
 
 #if LWIP_PERF
 #include "arch/perf.h"
diff --git a/func/lwip_intf/lwip-2.0.2/src/include/lwip/sockets.h b/func/lwip_intf/lwip-2.0.2/src/include/lwip/sockets.h
index 0f6e8be9c..4048f9a7c 100644
--- a/func/lwip_intf/lwip-2.0.2/src/include/lwip/sockets.h
+++ b/func/lwip_intf/lwip-2.0.2/src/include/lwip/sockets.h
@@ -38,7 +38,7 @@
 #define LWIP_HDR_SOCKETS_H
 
 #include "lwip/opt.h"
-#include "mem_pub.h"
+//#include "mem_pub.h"
 
 #if LWIP_SOCKET /* don't build if not configured for use in lwipopts.h */
 
